msi/release.sh

```diff
@@ -118,10 +118,35 @@ SCRIPT_PATH="$(cd "$(dirname "$0")" && pwd)"
 cd "$SCRIPT_PATH" ||
 die "Could not switch directory to $SCRIPT_PATH"
 
+# Generate the ReleaseNotes.html file
+test -f ../ReleaseNotes.html &&
+test ../ReleaseNotes.html -nt ../ReleaseNotes.md &&
+test ../ReleaseNotes.html -nt release.sh || {
+    # Install markdown
+    type markdown ||
+    pacman -Sy --noconfirm markdown ||
+    die "Could not install markdown"
+
+    (printf '%s\n%s\n%s\n%s %s\n%s %s\n%s\n%s\n%s\n' \
+        '<!DOCTYPE html>' \
+        '<html>' \
+        '<head>' \
+        '<meta http-equiv="Content-Type" content="text/html;' \
+        'charset=UTF-8">' \
+        '<link rel="stylesheet"' \
+        ' href="usr/share/git/ReleaseNotes.css">' \
+        '</head>' \
+        '<body class="details">' \
+        '<div class="content">'
+     markdown ../ReleaseNotes.md ||
+     die "Could not generate ReleaseNotes.html"
+     printf '</div>\n</body>\n</html>\n') >../ReleaseNotes.html
+}
+
```

Comment:
Sorry to leave this unanswered for such a long time. I would make only the `.css` location as referenced in the `.html` configurable. And for the preview, I would actually generate the `.html` in the top-level directory, for the installer in the `installer/` directory, for NuGet in the `nuget/` directory, and for MSI in the `msi/` directory. That way, we can keep the up-to-date check and everything will be quite a bit neater than it is now.